scheduler.py

```diff
@@ -1,128 +0,0 @@
-            ticker_count[clean_word] = ticker_count.get(clean_word, 0) + 1
-
-
-# Filters out the abbreviations that generally do not represent stock tickers
-def filter_dictionary(sorted_ticker_count, black_list):
-    clean_sorted_ticker_dict = {}
-
-    for key, value in sorted_ticker_count.items():
-        if key not in black_list:
-            clean_sorted_ticker_dict[key] = value
-        if len(clean_sorted_ticker_dict) == 10:
-            break
-
-    return clean_sorted_ticker_dict
-
-
-# Packages the top10 tickers and related financial information into dictionaries
-# and inserts them into the database with time stamps
-def insert_to_db(sorted_ticker_list, filtered_sorted_ticker_dict, ticker_dict):
-    toDB = []
-
-    now = datetime.now()
-
-    for i in range(10):
-        output_ticker = sorted_ticker_list[i]
-        output_count = filtered_sorted_ticker_dict[output_ticker]
-        stock_info = yf.Ticker(output_ticker).info
-        entry = {
-            "rank": i + 1,
-            "ticker": output_ticker,
-            "name": ticker_dict[output_ticker],
-            "industry": stock_info["industry"],
-            "count": output_count,
-            "previousClose": stock_info["previousClose"],
-            "fiftyDayAverage": stock_info["fiftyDayAverage"],
-            "averageDailyVolume10Day": stock_info["averageDailyVolume10Day"],
-            "timeStamp": now
-        }
-        toDB.append(entry)
-
-    print(toDB)
-
-    collection.insert_many(toDB)
-
-
-def main():
-    # Creates a combined NASDAQ and NYSE stock ticker dictionary
-    ticker_dict = {}
-    build_dictionary("Ticker Symbols/NASDAQ.txt", ticker_dict)
-    build_dictionary("Ticker Symbols/NYSE.txt", ticker_dict)
-
-    # Counts the appearances of tickers in the top500 hot posts in WSB
-    sorted_ticker_count = count_tickers(reddit, ticker_dict)
-
-    # Filters out the abbreviations that generally do not represent stock tickers
-    # Also cuts down the size of the dictionary to 10
-    filtered_sorted_ticker_dict = filter_dictionary(sorted_ticker_count, black_list)
-
-    # Gets the list of top 10 tickers from the filtered dictionary
-    sorted_ticker_list = list(filtered_sorted_ticker_dict.keys())
-
-    # Inserts top10 tickers and the related financial information into the database
-    insert_to_db(sorted_ticker_list, filtered_sorted_ticker_dict, ticker_dict)
-
-
-if __name__ == "__main__":
-    # Connects to database
-    load_dotenv()
-    client = MongoClient(os.environ['DB_URL'])
-    collection = client["findMyGME"]["stocks"]
-
-    # Creates an instance of the reddit object
-    reddit = praw.Reddit(client_id=os.environ['client_id'], client_secret=os.environ['client_secret'],
-                         user_agent=os.environ['user_agent'])
-
-    # A list of abbreviations that generally do not represent tickers.
-    black_list = ["A", "DD", "FOR", "CEO", "ALL", "EV", "OR", "AT", "RH", "ONE", "ARE", "VERY", "ON", "EDIT"]
```

Comment:
Create a constant on top of the file
TICKER_BLACKLIST = ["A", "DD", "FOR", "CEO", "ALL", "EV", "OR", "AT", "RH", "ONE", "ARE", "VERY", "ON", "EDIT"]